apps/tlon-mobile/src/screens/ProfileScreen.tsx

```diff
@@ -23,6 +24,16 @@ export default function ProfileScreen(props: Props) {
     props.navigation.navigate('EditProfile');
   }, [props.navigation]);
 
+  const [dmLink, setDmLink] = useState('');
+
+  useEffect(() => {
```

Comment:
Maybe this goes in `utils/branch` as `useDmLink`?

Reply:
Totally. Let me move.